src/utils/user_utils.js

```diff
@@ -21,7 +21,7 @@ export function displayUsername(
     user: UserProfile,
     teammateNameDisplay: Preferences.DISPLAY_PREFER_NICKNAME | Preferences.DISPLAY_PREFER_FULL_NAME
 ): string {
-    let name = '';
+    let name = General.UNKNOWN_USERNAME;
```

Comment:
This seems like a straightforward solution! @sudheerDev, can you add test coverage for this path?